phi/playground/playground.py

```diff
@@ -131,6 +136,15 @@ def agent_chat(body: AgentRunRequest):
                 run_response = cast(RunResponse, agent.run(body.message, stream=False))
                 return run_response.model_dump_json()
 
+        @playground_routes.post("/agent/rename")
```

Comment:
1. We should also get `session_id` in the body, otherwise dont know which session to rename
2. Change route to maybe `/agent/sessions/rename` as we're renaming the session ?